src/main/java/uk/gov/hmcts/reform/sscs/domain/DwpAddress.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.sscs.domain;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class DwpAddress {
+    private final Optional<String> line1;
+    private final Optional<String> line2;
+    private final Optional<String> line3;
+    private final Optional<String> postCode;
+
+    public DwpAddress(String line1, String line2, String line3, String postCode) {
+        this.line1 = Optional.ofNullable(StringUtils.stripToNull(line1));
+        this.line2 = Optional.ofNullable(StringUtils.stripToNull(line2));
+        this.line3 = Optional.ofNullable(StringUtils.stripToNull(line3));
+        this.postCode = Optional.ofNullable(StringUtils.stripToNull(postCode));
+    }
+
+    public DwpAddress(String line1, String line2, String postCode) {
+        this(line1, line2, null, postCode);
```

Comment:
does this mean that there will be a gap on the envelope when there is only 3 lines for an address? Might be better to have `this(line1, line2, null, postCode);` and then rename `postCode` to `line4`?